src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	private String isAdd;
+	private Tag tag;
+	private Person person;
+	
+	public Tagging(boolean add, Person p, Tag t) {
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.